packages/manager/modules/sign-up/src/form/sign-up.scss

```diff
@@ -0,0 +1,7 @@
+.required-field {
+  .oui-field__label::after {
+    content: ' *';
+    color: #c11b1b;
```

Comment:
Can you use css variables please ?
Here you can use `$ae-500`
And for the font-weight below, you could use `$jupiter-font-weight`